cli/src/commands/init.ts

```diff
@@ -1,12 +1,15 @@
 import type { Arguments, CommandBuilder } from "yargs";
 import { execSync } from "child_process";
 import { Options } from "yargs-parser";
-import { downloadFileIfNotExists } from "../utils/downloadBin";
+import { snakeCase } from "snake-case";
+import { downloadCargoGenerateIfNotExists } from "../utils/downloadBin";
 
 const path = require("path");
 export const command: string = "init [name]";
 export const desc: string = "Initialize a PSP project";
 
+const defaultProgramId = "Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS";
```

Comment:
What is the     --define program-id=${defaultProgramId} 
 Used for?

Reply:
It's used here https://github.com/Lightprotocol/psp-template/pull/1/files#diff-5ab73efa02b4d38fe93c54152638c6e112f6180d2900ecaedacc790e85d27f7bR12-R15

and then:

https://github.com/Lightprotocol/psp-template/pull/1/files#diff-0232306f21e881bf09e02d5acaec1ffc806ab5994d156514dc5ebe208184ea57R6
https://github.com/Lightprotocol/psp-template/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R5
https://github.com/Lightprotocol/psp-template/pull/1/files#diff-500a81c0a3f945b984072e2ac1d5e208b2ad158bc25d52ac502bb0129d909a9cR15
https://github.com/Lightprotocol/psp-template/pull/1/files#diff-500a81c0a3f945b984072e2ac1d5e208b2ad158bc25d52ac502bb0129d909a9cR18
https://github.com/Lightprotocol/psp-template/pull/1/files#diff-e80f9687cd11ce4e7de1e55ed48e3970cc7675fe016098743a33f6e246e4b42fR21
https://github.com/Lightprotocol/psp-template/pull/1/files#diff-9eb2487c4aa026b9064ed9c01a273f746fbebc9559e13309fe34518cc660c124R32

Reply:
got it, so its used for testing; was wondering if this will be relevant for when people start deploying this to public testnet